frontend/src/components/auth/StudentLogin.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import { PinInput, PinInputField, HStack, Button } from "@chakra-ui/react";
+import AuthWrapper from "./AuthWrapper";
+import { STUDENT_SIGNUP_IMAGE } from "../../assets/images";
+import { LeftArrowIcon } from "../../assets/icons";
+
+const StudentLogin = (): React.ReactElement => {
+  // For back button
+  const history = useHistory();
+
+  // Some CS
+  const pinWidth = "60px";
+  const pinHeight = "100px";
+  const margin = "10px 10px 90px 10px !important";
+  const fontSize = "30px";
+  const fontColor = "grey.300";
+
+  // Fields for AuthWrapper component
+  const title = "Student Login";
+  const subtitle = "Please enter your classroom's access code";
+  const image = STUDENT_SIGNUP_IMAGE;
+  const form = (
+    <>
+      <HStack>
+        <PinInput
+          autoFocus
+          focusBorderColor="blue.100"
+          placeholder=""
+          size="lg"
+          variant="filled"
```

Comment:
can you make the input controlled? (example here https://www.npmjs.com/package/@chakra-ui/pin-input under `controlled`)